node/cn/tracers/doc.go

```diff
@@ -22,5 +22,7 @@ Source Files
 
   - tracer.go  : implementation of Tracer
   - tracers.go : provides managing functions of tracers
+  - api.go     : implementation of tracer apis
+  - api_test.go: unit test of tracer apis
```

Comment:
We don't add explanations for test files.
```suggestion
```

Reply:
Reflected at https://github.com/klaytn/klaytn/pull/1581/commits/c2360ce46cb2c71560c2e0557bad56abed7af9bc